FlowCal/mef.py

```diff
@@ -265,27 +320,34 @@ def fit_beads_autofluorescence(fl_channel, fl_mef):
     # Initial guesses:
     # 0: slope found by putting a line through the highest two points.
     # 1: y-intercept found by putting a line through highest two points.
-    # 2: bead autofluorescence initialized to 100.
+    # 2: bead autofluorescence initialized using the first point.
     params[0] = (np.log(fl_mef[-1]) - np.log(fl_mef[-2])) / \
-                    (fl_channel[-1] - fl_channel[-2])
-    params[1] = np.log(fl_mef[-1]) - params[0] * fl_channel[-1]
-    params[2] = 100.
+                (np.log(fl_au[-1]) - np.log(fl_au[-2]))
+    params[1] = np.log(fl_mef[-1]) - params[0] * np.log(fl_au[-1])
+    params[2] = np.exp(params[0]*np.log(fl_au[0]) + params[1]) - fl_mef[0]
```

Comment:
I know our cytometer misbehaves with peaks up near the top end (you can sometimes see an explicit kink in the standards curve with the last peak if you don't manually throw it out), which undermines my faith in the top two peaks in general. That being said, usually that top peak should be thrown out if necessary by a user who is paying attention, and I still think the fitting winds up at the same solution, so I think you're probably right that the end result is good enough.